tornado/web.py

```diff
@@ -466,7 +466,8 @@ def get_arguments(self, name: str, strip: bool = True) -> List[str]:
         # Make sure `get_arguments` isn't accidentally being called with a
         # positional argument that's assumed to be a default (like in
         # `get_argument`.)
-        assert isinstance(strip, bool)
+        if not isinstance(strip, bool):
```

Comment:
I mostly agree with getting rid of the `assert` statement, but this seems like a case where it's warranted - if it's relevant it'll fire all the time so you'll see it in development (or even better, a type checker can catch it statically), and then you can disable the assertion in production with `-O`